index.ts

```diff
@@ -1,30 +1,11 @@
 interface Recommendation {
-  title: string;
-  category: string;
   uri: string;
+  title: string;
   imageUri: string;
+  category: string;
 }
 
-const recommendations: Recommendation[] = [
-  {
-    title: 'Test',
-    category: 'Fasting',
-    uri: 'https://cru.org',
-    imageUri: 'https://via.placeholder.com/300x200',
-  },
-  {
-    title: 'Second Test',
-    category: 'Fasting',
-    uri: 'https://cru.org',
-    imageUri: 'https://via.placeholder.com/400',
-  },
-  {
-    title: 'Third Test',
-    category: 'Fasting',
-    uri: 'https://cru.org',
-    imageUri: 'https://via.placeholder.com/150',
-  },
-];
+const apiUri = 'https://cru-content-based-filtering.s3.amazonaws.com/';
```

Comment:
Is this the same on prod and stage?

Reply:
As of now I only know of one BigQuery table and Brian only created me one S3 bucket. I was debating forcing the consumer to specify this in the options. Idk if there's a need for it now 🤷‍♂️ Maybe we need to discuss environments more...

Reply:
That would be a good discussion with Andrea/Phyllis.